src/bench/random.cpp

```diff
@@ -378,12 +391,13 @@ bench_thread(PriorityQueue *pq,
                              settings.size - thread_id * slice_size : slice_size;
     for (int i = 0; i < initial_size; i++) {
         uint32_t elem = keygen.next();
+        uint32_t value = keygen.next();
```

Comment:
Is it beneficial to add a separate value? I reused elem since the value seems irrelevant and it just adds keygen overhead. Same below.